jme3-lwjgl3/src/main/java/com/jme3/system/lwjgl/LwjglWindow.java

```diff
@@ -276,11 +276,16 @@ public void invoke(final long window, final boolean focus) {
             }
         });
 
-        // Center the window
         if (!settings.isFullscreen()) {
-            glfwSetWindowPos(window,
+            if (settings.getCenterWindow())
+            {
+                // Center the window
+                glfwSetWindowPos(window,
                     (videoMode.width() - settings.getWidth()) / 2,
                     (videoMode.height() - settings.getHeight()) / 2);
+            } else {
+            	glfwSetWindowPos(window,settings.getWindowXPosition(), settings.getWindowYPosition());
```

Comment:
This line of code should be indented at least 3 spaces further.

Reply:
What do you think bob0bob?

Reply:
What are your default tabs.  The tabs are right on my system.